doc/input-output-reference/src/overview/group-thermal-zone-description-geometry.tex

```diff
@@ -3549,101 +3549,63 @@ \subsubsection{Surface Window Gap Convective Heat Transfer Energy {[}J{]}}\label
 
 For an airflow window, the forced convective heat flow from the gap through which airflow occurs. This is the heat gained (or lost) by the air from the glass surfaces (and between-glass shading device surfaces, if present) that the air comes in contact with as it flows through the gap. If the gap airflow goes to the zone indoor air, the gap convective heat flow is added to the zone load. Applicable to exterior windows only.
 
-\subsubsection{Surface Window Heat Gain Rate {[}W{]}}\label{surface-window-heat-gain-rate-w}
+\subsubsection{Surface Window Net Heat Transfer Rate {[}W{]}}\label{surface-window-net-heat-transfer-rate-w}
 
-\subsubsection{Surface Window Heat Gain Energy {[}J{]}}\label{surface-window-heat-gain-energy-j}
+\subsubsection{Surface Window Net Heat Transfer Energy {[}J{]}}\label{surface-window-net-heat-transfer-rate-j}
```

Comment:
Update this section use actual output variable names for the components, add missing components, and delete duplicate list of components. Reviewers, please read these doc changes carefully to see if they make sense.